### PR TITLE
Add: Instructions to preview changes on netlify

### PR DESCRIPTION
**Pull request by RaiVaibhav**
Instructions to preview changes on a PR on netlify

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,6 +34,13 @@ In general we'd like to ask you to please target the master branch with your pul
 We recommend creating a new branch for each feature task/bug since it makes easier to edit
 commits in future if needed.
 
+After submitting a pull request preview your changes by going through the checks section in
```
Comment (astronomersiva) [inline CONTRIBUTING.md]: `pull request, preview`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -34,6 +34,13 @@ In general we'd like to ask you to please target the master branch with your pul
...
+last of the pull request page and click `Details` of the `netlify/cranky-borg-621833/deploy-preview` check.
```
Comment (astronomersiva) [inline CONTRIBUTING.md]: There is a separate `Checks` tab. We can talk about that instead of this approach.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -34,6 +34,13 @@ In general we'd like to ask you to please target the master branch with your pul
...
+
+![Screenshot from 2019-03-31 22-25-01](https://user-images.githubusercontent.com/22278438/55292083-ff843400-5403-11e9-8bbc-1f92c3733627.png)
```
Comment (astronomersiva) [inline CONTRIBUTING.md]: Alt Names should be meaningful.

File: CONTRIBUTING.md (hunk 2 above)
Comment (RaiVaibhav) [reply CONTRIBUTING.md]: Yes wrt to the debugging and review going to the checks tab and previewing every page changed is best, also from index link anyone can also preview any other page also, so I am also in favour of going through the checks tab and previewing every tab

Should I change the screenshot and update the instructions?

Comment (astronomersiva) [reply CONTRIBUTING.md]: Yes, we can update this text to tell people to click that tab and access the changed files.

**Comment by astronomersiva** [comment]
Thanks for working on this!

**Pull request merged**